wfl/generate/md/__init__.py

```diff
@@ -136,19 +137,30 @@ def _sample_autopara_wrappable(atoms, calculator, steps, dt, temperature=None, t
 
         if temperature is not None:
             # set initial temperature
+#            print("temperature ", temperature)
             MaxwellBoltzmannDistribution(at, temperature_K=temperature[0]['T_i'], force_temp=True, communicator=None)
-            Stationary(at, preserve_temperature=True)
+#            Stationary(at, preserve_temperature=True)
 
         stage_kwargs = {'timestep': dt * fs, 'logfile': logfile}
 
-        if temperature_tau is None:
+        if temperature_tau is None and friction is None:
             # NVE
             if pressure is not None:
                 raise RuntimeError('Cannot do NPH dynamics')
             md_constructor = VelocityVerlet
             # one stage, simple
             all_stage_kwargs = [stage_kwargs.copy()]
             all_run_kwargs = [ {'steps': steps} ]
+
+        elif temperature_tau is None and friction is not None:
```

Comment:
I don't know how I can relate `temperature_tau` and `friction`.
It was easy way to include Langevin dynamics into the MD script. 
I think larger modification might be needed in order to make it to specify which thermostat to be used.

Reply:
Well, I definitely don't think we should have two different arguments that aren't can't be specified at the same time and have it behave unexpectedly if you pass both. tau = 1 / friction.

Reply:
Following some advice from Elena, `__init__.py` in md has been restructured by adding new argument `integrator`. This argument has to be either `default` or `langevin` and if one chooses langevin, it will get friction value from `1/temperature_tau`. However, I think it could be more intuitive for users to use `temperature_tau` rather than `1/temperature_tau`.

Reply:
Why `"default"`, rather than `"berendsen"` (or maybe the full ASE name `"NVTBerendsen"`), and just make that the default value?